src/api/PDFPageOptions.ts

```diff
@@ -51,15 +51,19 @@ export interface PDFPageDrawSVGOptions {
   opacity?: number;
   borderColor?: Color;
   borderOpacity?: number;
+  dashArray?: number[];
+  dashPhase?: number; 
```

Comment:
Can we call these options `borderDashArray` and `borderDashPhase`? This maintains the convention of prefixing all options related to the border with `border...`. And, of course, this will also need to be changed most of the other places these options are used.

Reply:
You're right, done!